### DISCUSSION TITLE
Apps install

### DISCUSSION THREAD

**Discussion by Silence714** [dat-linux/community#43] [Issues]
I've installed DAT Linux on Vm, network is fine and I can open WEB on my Browser . But It doesn't install the Apps .. I attach the screenshot.
![Screenshot (2)](https://github.com/dat-linux/community/assets/152275289/389b726a-68f8-4b6c-a7aa-fd52bedaecf9)

**Comment by dat-linux** [comment]
Google server is pinged to check for internet connection. Repeat using command:
```
. /usr/local/bin/datlinux/dat-ping
check_internet || check_internet_err_msg
```
If returns error, then google.com cannot be seen from `wget`.
